main.go

```diff
@@ -36,7 +37,8 @@ type options struct {
 
 func main() {
 	var opts options
-	if _, err := flags.Parse(&opts); err != nil {
+	tables, err := flags.Parse(&opts)
```

Comment:
Instead of command line argument, can we use command line option to specify table_name like `--tables=t1,t2,...`?

Unlike mysqldump, this tool uses options to get even required parameters.